spec/unit/module_spec.rb

```diff
@@ -91,6 +91,17 @@
           expect(obj).to be_a_kind_of(R10K::Module::Base)
         }.not_to raise_error
       end
+      describe 'the deploy_spec setting' do
+        it 'sets the deploy_spec instance variable' do
+          obj = R10K::Module.new(name, '/modulepath', options.merge({deploy_spec: true}))
+          expect(obj.instance_variable_get("@deploy_spec")).to eq(true)
+        end
+        it 'can be overridden by the overrides map' do
+          options = options.merge({deploy_spec: false, overrides: {modules: {deploy_spec: true}}})
+          obj = R10K::Module.new(name, '/modulepath', options)
+          expect(obj.instance_variable_get("@deploy_spec")).to eq(true)
+        end
```

Comment:
Would it be worth testing that `@deploy_spec` is false if not passed as an option? Or does it stay `nil`?

Reply:
For this test, it would just be `nil`; it didn't seem terribly important test.